### PR TITLE
(Ready) Flare (70009)

### PR DESCRIPTION
**Pull request by PummelsTL**

### CODE REVIEW

**Review by Giratena — CHANGES_REQUESTED** [review]
Translation looks good from a once over Pummels! Nice work ^^

Just a quick heads up for this and future PRs to keep in mind the punctuation around hearts:

> Any ❤, must be prefixed with puncuation. Eg: ".❤", "!❤", "?❤", "~❤", or "...❤"

This is something that I think should be consistent across the translation so I'd like to really push for it. Feel free to do a global find and replace for it if you need to ^^

**Comment by PummelsTL** [comment]
Added punctuation for the hearts.

**Review by Giratena — CHANGES_REQUESTED** [review]
Not bad @PummelsTL! Good accurate translations for the most part. Nice work!

I had a look and noticed some things with formatting. Please take a look!

I also noticed there appears to be a lack of attention towards laughing sfx, choosing to translate them all as a blunt "Ahaha." I've pointed these out for you to have a look at. I hope it helps. 

I know it might seem trivial, but it can potenitally muddle the intended voice of a Japanese character if not done correctly.

Hope this review helps :D

File: CSV/Language/English/Card/70009.txt [hunk 1]
```diff
@@ -2,238 +2,240 @@ default
...
+くすくす❤おっぱいでズリズリってしてあげたら、\nとっても情けない顔で射精しちゃったわね❤\n谷間にこんなに出してスケベなんだから❤,Ahaha.❤ You blew your load with such a miserable face to my boobjob.❤ You must be a total perv since you cummed this hard in my breasts.❤
+脚でいじめてあげたら、アヘアへ言いながら出しちゃったわ❤\nわたしにバカにされながらいじめられて、気持ちよかったんだ❤\nマゾなモンスターね。くすくす❤,You blew your load while moaning hard under my feet.❤ Was it that amazing to have me trample and make fun of you? You're such a masochistic monster.❤ Ahahaha.❤
+……っ！❤,...!❤
+もしかしてアンタ、またわたしでオナニーしちゃうの？前回それで負けてるくせに、学習能力ないのかしら❤,Are you masturbating to me again? You know; that's how you lost last time. Are you so stupid that you can't learn from your mistakes?
+あぁ それとも、"わたしのおっぱいには勝てない"ってことを学習しちゃったの？ふふ❤,Or perhaps it's not that you haven't learned but... You've learned how impossible it's to win against my breasts? Ahaha.❤
+う、うぅ…❤,Uh... Uuuh...❤
+あっは❤ばかにされてしごくスピード速くなってるし❤男にとってはそんなに我慢できないものなのかな、おっきいおっぱいって❤,Ahaha❤ Your hand sped up when I made fun of you.❤ Do men find huge breasts like mine just that irresistible?
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:111]: Fixed.

File: CSV/Language/English/Card/70009.txt [hunk 2]
```diff
@@ -2,238 +2,240 @@ default
...
+ちょっと待っ、今は…っ！❤,Wait... Right now; I'm...!
+だーめ❤待ってあげない❤,Nope!❤ I won't wait even a moment.❤
+うふふ❤口ではそんなこと言ってても、\nわたしにおっぱい動かされたら\nすぐにチンポガチガチにしちゃってるじゃない❤,Ahaha.❤ Even when you told me to stop; your penis got harder when I started moving again.❤
+理性では止めてほしいって思っても、\n女の子にえっちなことされたら体はどんどんエナジー放出しちゃう❤\nしかも、それが気持ちよくてたまらない❤,Your mind is telling to stop; but your body is wanting more lewd things; to offer more mana to me.❤ My breasts just feel that irresistibly good.❤
+サキュバスだっていることだし、\nこの調子なら女の子のわたしの方が\n男よりもずっと強くなれるかも❤,If I keep doing this; maybe I can become stronger than any man.❤ That's how it works for succubi; right?❤
+うぅ、そんなぁ…❤,Uuuh... That's...❤
+それに、男からエナジーを吸い取れるのは\nサキュバスだけの専売特許じゃないわ❤,It's not like draining energy from men is only limited to succubi.❤
+アンタも知ってるでしょうけど、\nカードマスターの女の子なら精液から魔力を補給できる❤\nもちろん サキュバスの効率には敵わないけれど……,As you surely know; female cardmasters can also replenish mana with semen.❤ It's quite inefficient compared to how succubus succubi do it with energy drain...
+それなら、こうしちゃえばいいのよ❤,But the inefficiency shouldn't a problem if I do this.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:229]: I agree.

File: CSV/Language/English/Card/70009.txt [hunk 3]
```diff
@@ -2,238 +2,240 @@ default
...
+わたしのムチムチのおっぱいで挟んであげれば、\nみーんな簡単にイっちゃうもの❤\nそしたら爆破に耐えられる体力を持ってたって無駄❤,No one can help but miserably spill their semen when I squeeze their penises between my soft breasts.❤ Resisting is futile even if they're strong enough to withstand my explosions.❤
+この爆乳に挟まれて、\nチンポの気持ちよさを我慢できる男なんていないわ❤\n大事なエナジーをわたしに抜き取られちゃうしかないのよ❤,There isn't a single man who can endure the pleasures of my explosive breasts with their penis.❤ Giving your mana to me is your only option.❤
+ほらほら、アンタももう限界でしょ？\nわたしのパイズリで、貴重な魔力\n思いっきりぶちまけちゃいなさい❤,Now; you're about to hit the limit; right? Go ahead and do it.❤ Give away all your precious mana to my boobjob.❤
+あ、あ、うぁ……っ❤,Ah... Ah... Aah...❤
+うぁ、はぅ、う…っ❤,Uh... Ah... Uuuh...❤
+ふふ、出た出た❤\nチンポをおっぱいで捕まえちゃえば、男なんてわたしの思うままよ❤,Ahaha. There it is.❤ Men have no choice but to do as I say once I have their penis trapped between my breasts.❤
+う～ん おいしい❤やっぱり、アンタって\n良質な魔力を持ってるみたい❤もっともっと搾りとってあげるから、\nわたしにぜんぶ捧げちゃいなさい❤,Mmmh... Delicious.❤ You've got some fine-quality mana❤ I'll keep draining you for more; so make sure to offer it all to me.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:220]: I agree. Added suggestions.

File: CSV/Language/English/Card/70009.txt [hunk 4]
```diff
@@ -2,238 +2,240 @@ default
...
+別にいいじゃない❤どうせ一回誘惑に負けちゃったら、\n男はもう気持ちよくなることで\n頭の中がいっぱいになっちゃうんでしょ❤,And it's not like you mind; right?❤ After all; now that you've fallen to the temptation once; pleasure should be the only thing in your mind.❤
+そんなこ……うっ、あ、はぅ…❤,That's not... Uuh... Aaah!❤
+耐えようと思っても感じちゃってしかたがないはずよ❤\nこれが、ピンチの時に使う わたしの本気パイズリ❤,Even if you try to resist; you can't help the pleasure taking over your body.❤ This is the serious boobjob that I use when I'm in a pinch.❤
+ただでさえおっぱいに弱い男にわたしがこれをしてあげれば、\n１００パーセント射精させられるわ❤,I have a 100% record of making every man who's weak to breasts cum with this.❤
+ガチガチのチンポもみくちゃにされて\nおっぱいがぬちゅぬちゅ絡みついてきて、\n男には たまらないわよね❤,Men just can't get enough of it when I squeeze and knead their penis with my soft breasts like this.❤
+ほら、またイっちゃいなさい❤\n敵マスターにどんどんエナジー出しちゃえ❤,Now; cum again.❤ Give your energy to the opposing cardmaster.❤
+うぁ、あ、あぅぅ…❤,Uh... Ah... Aaaah...❤
+はい、イっちゃった❤\nね？一回チンポ挟んじゃえば、\nうぶな男の子なんて何回でも連続射精させられるわ❤,There it is.❤ You know? It's a trifling thing for me to make naive boys like you cum several times in a row once I have your penis between my breasts.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:239]: Changed to: "And there: You spilled your load.❤ Did you know?"

File: CSV/Language/English/Card/70009.txt [hunk 5]
```diff
@@ -2,238 +2,240 @@ default
...
+はぅ、うぅあ…❤,Ah... Uuuh...❤
+それに、今アンタにもしてあげてるパイズリ❤\n誘惑しても手強い大人はもちろんいるけど、\nその時はこうやって直接エナジー抜きとっちゃえば問題なしよ❤,There's also the matter of boobjobs–like the one I'm giving you right now.❤ Naturally; some strong men can resist my charm and amazing figure; but that's no longer an issue when I'm directly draining their mana like this.❤
+わたしのムチムチのおっぱいで挟んであげれば、\nみーんな簡単にイっちゃうもの❤\nそしたら爆破に耐えられる体力を持ってたって無駄❤,No one can help but miserably spill their semen when I squeeze their penises between my soft breasts.❤ Resisting is futile even if they're strong enough to withstand my explosions.❤
+この爆乳に挟まれて、\nチンポの気持ちよさを我慢できる男なんていないわ❤\n大事なエナジーをわたしに抜き取られちゃうしかないのよ❤,There isn't a single man who can endure the pleasures of my explosive breasts with their penis.❤ Giving your mana to me is your only option.❤
+ほらほら、アンタももう限界でしょ？\nわたしのパイズリで、貴重な魔力\n思いっきりぶちまけちゃいなさい❤,Now; you're about to hit the limit; right? Go ahead and do it.❤ Give away all your precious mana to my boobjob.❤
+あ、あ、うぁ……っ❤,Ah... Ah... Aah...❤
+うぁ、はぅ、う…っ❤,Uh... Ah... Uuuh...❤
+ふふ、出た出た❤\nチンポをおっぱいで捕まえちゃえば、男なんてわたしの思うままよ❤,Ahaha. There it is.❤ Men have no choice but to do as I say once I have their penis trapped between my breasts.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:219]: Changed to "Ahaha! You blew your load.❤"

File: CSV/Language/English/Card/70009.txt [hunk 6]
```diff
@@ -2,238 +2,240 @@ default
...
+またいつでも再戦待ってるわね！,You can for a rematch anytime!
+あなたのターンからね。,You go first.
+わたしのターンからね。,I go first.
+ドロー！,Draw!
+ドローするわ。,I draw a card.
+行くわよ、{使用カード}を上級召喚！,Here I go: Summon {使用カード}!
+{使用カード}を召喚。,Summon {使用カード}.
+ううっ、あうう・・・❤,Uuuh... Auuuh...❤
+あは、アンタ復活したのにチンポギンギンじゃない❤\n自分から負けちゃっただけあって\n理性がゴミみたいに弱っちいのね❤,Ahaha. Your penis remains rock-hard even after getting revived.❤ Your mind must be weak as trash since you actually chose to lose to me.❤
+こんな風に罵られてもうっとりした顔のままだし、重症だわ❤\n何もできないでくの坊のにはさっさと退場願うとしましょう。\nくすくす❤,And you're even watching me with enraptured eyes even though I'm making fun of you.❤ You've got some serious issues.❤ Idiots who can't do anything should leave the field already.❤ Ahaha.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:38]: I chose to remove this laugh at the end altogether.

File: CSV/Language/English/Card/70009.txt [hunk 7]
```diff
@@ -2,238 +2,240 @@ default
...
+見なさい。わたし胸大きいでしょ？これ急に大きくなったから服のサイズが間に合わなくってパツパツなの。,Look. My breasts are huge; right? I've gone through such a huge growth spurt that my shirt barely fits me.
+いまもまだおっきくなっている最中でしょっちゅうボタンが飛んでっちゃうのよ。めんどうったらないわ。,And they still haven't stopped growing. I'm constantly having buttons popping off. It's quite troublesome.
+でもそのおかげで最近の戦績は向上中❤だって男ってこれにあんまりに弱いんだもの❤,But it's thanks to these breasts that my win rate has seen explosive growth recently.❤ Men are just that hopeless before such a massive bosom.❤
+あんたもそうよね？ほら、パイズリしてあげるからさっさとわたしに白旗あげちゃいなさい❤くすくす❤,You're the same; right? I'll give you a boobjob; so show me your white flag already.❤ Ahaha.❤
+谷間のボタンを外してっと…。\nアンタのチンポ、パイズリで気持ちよくしてあげる❤,I'll open a button on my shirt... And make you feel good with a boobjob.❤
+見て？丸ごと飲み込んじゃったわ❤\nおっぱいに完全に埋もれてチンポが見えない❤\nなんだかこうやって捕まえると優越感があるわよね♪,Look; your penis went in all the way to the base.❤ I've entirely hidden it between my breasts.❤ It tickles my sense of superiority when I've got you pinned like this♪
+おっぱいをズリズリ動かして、\n足腰立たなくなるまで気持ちよくしてあげる❤\n覚悟しなさい。ふふふ❤,Now; I'll start moving and make you feel so good that your legs give in.❤ Get ready for it. Ahahaha.❤
```
[the remark this answers is not in the document]
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:83]: I think this is fine as is.

File: CSV/Language/English/Card/70009.txt [hunk 8]
```diff
@@ -2,238 +2,240 @@ default
...
+ほぉら、おっぱいよ❤これがたまらなかったんでしょう？❤,Hey; look here: These are my breasts.❤ You find them completely irresistible; right?
+……！！❤,...!❤
+あはは、ちょっと強調してあげたらしごくスピード速くなった❤,Ahahaha! I only pushed them up a little; and your hand sped up.❤
+アンタたちってこれに弱すぎよね❤まぁ、わたしのおっぱいってサキュバスと比べてもなかなかの大きさだもの❤,Honestly; you men are way too weak to breasts.❤ However... I guess mine are so huge that I might even beat a succubus in terms of size.❤
+男がこれを気にしないっていう方が難しい話なのかしら❤,Perhaps it would be asking the impossible for you not to pay attention to such huge breasts in front of you?❤
+今回で何人目になるのかな、バトル中なのに わたしの体にメロメロになっちゃった男の人数❤,I wonder how many times does this make it now? There have been so many men who have become charmed by my body while I've dueled them.❤
+さ、アンタの目の前でおっぱい たっぷり見せつけてあげるから、しこしこ我慢できなかったチンポから白いのいっぱい出しちゃいなさい❤,Now... I'll stick my breasts right in front of your eyes; so keep stroking until your white stuff spills out.❤
+思いっきりかけていいわよ、精子ぶちまけて負けちゃえ❤,You can even shoot your load all over them.❤ Spill your semen and lose.❤
+あ、うっ、うぅ…っ❤,Ah... Uuuh... Uuuh❤
+あーあー、わたしに促されるままイっちゃって❤気持ちよかったかしら？うふふ❤,Aa-ah. You blew your semen just as I told you to.❤ Did it feel good? Ahaha.❤
```
Comment (Giratena) [inline CSV/Language/English/Card/70009.txt:138]: Ah-ah → Oh dear,
Ahaha -> Ufufu or *giggle*

I think あーあー has more of a condesending vibe than just a laugh.

File: CSV/Language/English/Card/70009.txt [hunk 9]
```diff
@@ -2,238 +2,240 @@ default
...
+あ…、あぅ…❤,Ah... Aaah...❤
+あらあら モンスターくん？敵マスターの前でチンポしごき始めちゃって、どうしたのかしら。,Oh my... Mr. Monster? Why did you suddenly start masturbating to your master's opponent?
+わたしの体がえっちすぎて、理性が溶けちゃった？まったく、今はバトル中なのよ？❤,Is my body so irresistibly sexy that it made your mind melt? Good grief. We're in the middle of a duel; you know?❤
+うぅ…❤,Uuh...❤
+うふふ、君たちっていつもそうね❤立派な剣術や拳、魔法を持っているのに、敵の女の子がえっちだとすぐに惑わされちゃう❤,Ahaha. This is how it's for all you men.❤ Even if you're skilled with swords; fists; or magic; you can't help but get charmed the moment you face a sexy girl.❤
```
Comment (Giratena) [inline CSV/Language/English/Card/70009.txt:143]: Ufufu here or Ehehe etc

File: CSV/Language/English/Card/70009.txt [hunk 10]
```diff
@@ -2,238 +2,240 @@ default
...
+わたしを追い詰める実力を持ってる男に手加減なんてしてあげないわ❤ "イかせる"パイズリで、悶えちゃいなさい❤,I won't hold anything back since you managed to corner me.❤ Writhe as I make you cum with my boobjob.❤
+～～～～～っ！！❤,...!!!❤
+声にならないくらいきもちいい？❤男を射精させることだけ考えた、わたしの本気パイズリよ❤,Does it feel so good that you can't speak? This is the serious boobjob that I've come up with to make men cum as soon as possible.❤
+おっぱいでチンポもみくちゃにして、一瞬で精液 吐き出させるの❤,By tightly kneading your penis with my breasts like this; I can make you blow your load in an instant.❤
+抵抗なんて許さないわ❤すけべ心で勃起したチンポ ズリズリされて、射精で逆転負けしちゃいなさい❤,I won't give you any chance to resist.❤ Lose as I turn the tables on you by making you cum with my breasts.❤
+う、あ、うぁっ、あ……っ❤,Uh... Ah... Aah... Ah...❤
+あは、出した出した❤瞬殺❤わたしに高速パイズリされて、我慢できる男はいないわ❤,Ahaha! You blew your load!❤ It really was an instant finish.❤ There isn't a single man who can endure my all-out boobjob.❤
+おっぱいに気をとられて油断した方が悪いのよ、すけべ❤,You can only blame yourself for getting distracted by my breasts. You pervert.❤
+あーあー、自分でしこって出しちゃった❤ね、わたしの体がそんなにえっちだったの？うふふ❤,Aa-ah. You really went and stroked yourself all the way.❤ Was my body that irresistibly sexy? Ahahaha.❤
```
Comment (Giratena) [inline CSV/Language/English/Card/70009.txt:186]: Same as line 138

**Review by Giratena — CHANGES_REQUESTED** [review]
One major change here and I think that's it.

I still notice the translations of simply "Ahaha." do take away from some of the varied Japanese laughs. I'd have a look at those again if you have a change. After your next changes I'll merge ^^

File: CSV/Language/English/Card/70009.txt (hunk 8 above)
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:138]: There is no "Ah-ah" laugh here but an "Aa-ah" sound that carries a condescending tone.
I can't see changing the laugh changing anything.

File: CSV/Language/English/Card/70009.txt [hunk 11]
```diff
@@ -2,238 +2,240 @@ default
...
+ランクが上でも関係ない、\nこの間と同じようにまたわたしが勝つんだから！\nさあバトルよ！,It doesn't matter if you're higher rank or not! I'm going to win just like before! Now; let's duel!
+ううっ、強い…。\nでもわたし諦めない、次やるときは勝つんだから！\nまたやりましょう！,Uuh... You're so strong. But I'm not giving! I'll win next time! Mark my words!
+ランクが上の男でも射精しちゃうわたしのカラダ、\nアンタには刺激が強かったかもしれないわね❤\nふふ、エナジーごちそうさま❤,Even higher rank men can't help spilling semen to my body; so perhaps it's a bit too stimulating for a low-rank like you?❤ Ahaha. Thanks for the energy❤
+へへん、どう？わたしの炎デッキの力、すごいでしょ！,Ehehe. Well? The power of my flame deck is amazing; right?
+楽しいバトルだったわ。またいつでも再戦待ってるわね。,It was a good duel. You can come for a rematch anytime.
+ううっ、さすがに強いわね…。\n上のランクに行っただけあるわ…。悔しいぃ…。,Uuh... So strong. It's no wonder why you made it to the next rank... How frustrating...
+エナジーごちそうさま❤ランクが上でも、\nわたしの体で精液どぴゅどぴゅしちゃうのは変わらなかったわね❤,Thanks for the energy❤ Despite making it to the next rank; there's still no changing the fact that you can't help spilling your semen to my body❤
+やったぁ！上のランク相手でも勝てた！\nわたしの炎デッキの力、すごいでしょ！,Yay! I won against a higher-rank opponent! My flame deck is amazing!
```
Comment (Giratena) [inline CSV/Language/English/Card/70009.txt:28]: I think "Fire" deck might be a better name, for "Fire Attribute" etc

File: CSV/Language/English/Card/70009.txt [hunk 12]
```diff
@@ -2,238 +2,240 @@ default
...
+またいつでも再戦待ってるわね！,You can for a rematch anytime!
```
Comment (Giratena) [inline CSV/Language/English/Card/70009.txt:29]: -> Feel free to rematch again anytime!

File: CSV/Language/English/Card/70009.txt (hunk 8 above)
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:138]: あーあー -> Now; look at this.

File: CSV/Language/English/Card/70009.txt (hunk 10 above)
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:186]: Aa-ah. -> Now; look at you.

File: CSV/Language/English/Card/70009.txt (hunk 12 above)
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:29]: -> You can ask me for a rematch anytime!

File: CSV/Language/English/Card/70009.txt (hunk 11 above)
Comment (PummelsTL) [reply CSV/Language/English/Card/70009.txt:28]: Changed instances of flame to fire/fire-attribute.

**Review by Giratena — APPROVED** [review]
Nice work! Will approve and merge ^^
Thank you very much @PummelsTL!

**Pull request merged**